CONTRIBUTING.md

```diff
@@ -71,6 +71,31 @@ npm run test
 npm run test -- --watch
 ```
 
+### Debug
+
+There is a `debug` flag available on the configuration for `SearchDriver` and `SearchProvider`.
+
+```jsx
+<SearchProvider config={
+  debug: true
+  //...
+}>
+```
+
+Setting this to `true` will make the `driver` available globally on window. This will allow you to programmatically
+execute actions in the browser console which is helpful for debugging.
+
+```js
+window.driver.addFilter("states", "California", "all");
+```
+
+This will also log actions and state updates as they occur to the console in the following form:
+
+```
+Action {Action Name} {Action Parameters}
```

Comment:
Let's prefix these messages with "Search UI: "?
It will help users to distinguish these messages with Redux state and action updates and will allow easier filtering.